ntbs-service/Pages/Notifications/Overview.cshtml

```diff
@@ -13,10 +13,12 @@
 
 <partial name="./OverviewPartials/_PatientOverviewPartial"/>
 
-<partial name="./OverviewPartials/_EpisodeOverviewPartial"/>
+<partial name="OverviewPartials/_HospitalDetailsOverviewPartial"/>

```

Comment:
Have you verified that printing is still nice with the new additions? (as per a recent addition to the PR checklist)